packages/overlay/imperative-api.md

```diff
@@ -84,6 +84,135 @@ Some Overlays will always be passed focus (e.g. modal or page Overlays). When th
 
 The `trigger` option accepts an `HTMLElement` or a `VirtualTrigger` from which to position the Overlay.
 
+-   You can import the `VirtualTrigger` class from the overlay package to create a virtual trigger that can be used to position an Overlay. This is useful when you want to position an Overlay relative to a point on the screen that is not an element in the DOM, like the mouse cursor.
+
+### Using a virtual trigger
+
+```html-live
+<style>
+    #root {
+        position: relative;
+        width: 100%;
+        height: 20vh;
+        background-color: var(--spectrum-global-color-gray-100);
```

Comment:
Let's do this without `-global-` vars as they are deprecated. Maybe `--spectrum-gray-100`?

Reply:
oops yes!!